mor/pyfeelpp-mor/feelpp/mor/__init__.py

```diff
@@ -1,6 +1,7 @@
 import feelpp
 from ._mor import *
 from ._toolboxmor import *
+from ._pbdw import *
```

Comment:
may be we should have pbdw within its own separate module instead
it should probably not be loaded by default when we load pyfeelpp-mor

Reply:
should be in separate module